spec/models/manageiq/providers/workflows/automation_manager/workflow_instance_spec.rb

```diff
@@ -134,8 +135,8 @@
             "FirstState" => {
               "Type"        => "Succeed",
               "Credentials" => {
-                "username" => "$.username",
-                "password" => "$.password"
+                "username.$" => "$.username",
+                "password.$" => "$.password"
```

Comment:
yea. this is beautiful.

wish the credentials was a little more similar

Reply:
This was actually from a separate PR that I based this one on https://github.com/ManageIQ/manageiq-providers-workflows/pull/49